src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCacheKeyProvider.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-
-namespace Microsoft.AspNetCore.ResponseCaching
-{
-    public interface IResponseCacheKeyProvider
```

Comment:
There is discussion with whether it should be string based at all, and hashing of keys. I expect the interface to change dramatically anyway when we revisit.